lib/wordmove/logger.rb

```diff
@@ -2,40 +2,49 @@ module Wordmove
   class Logger < ::Logger
     MAX_LINE = 70
 
+    def initialize(device, strings_to_hide = [])
+      super(device, formatter: proc { |_severity, _datetime, _progname, msg|
+        if strings_to_hide.empty?
+          msg
+        else
+          "\n#{msg}\n".gsub(Regexp.new(strings_to_hide.join('|')), '[secret]')
+        end
+      })
+    end
+
     def task(title)
       prefix = "▬" * 2
       title = " #{title} "
       padding = "▬" * padding_length(title)
-
-      puts "\n" + prefix + title.light_white + padding
+      add(INFO, prefix + title.light_white + padding)
```

Comment:
woha! great catch! we were `put`ting to stdout anyway, even if the logger devise was a file!

Reply:
Moreover the custom formatter wouldn't have worked w/o using `add` method...